packages/babel-helpers/src/helpers/regeneratorRuntime.ts

```diff
@@ -115,44 +176,35 @@ export default function _regeneratorRuntime() {
     Generator.prototype =
       Object.create(IteratorPrototype));
   GeneratorFunction.prototype = GeneratorFunctionPrototype;
-  defineProperty(Gp, "constructor", {
-    value: GeneratorFunctionPrototype,
-    configurable: true,
-  });
-  defineProperty(GeneratorFunctionPrototype, "constructor", {
-    value: GeneratorFunction,
-    configurable: true,
-  });
-  GeneratorFunction.displayName = define(
-    GeneratorFunctionPrototype,
-    toStringTagSymbol,
-    "GeneratorFunction",
-  );
```

Comment:
We no longer set `GeneratorFunction.displayName`, but I think that's ok.

Reply:
I think this is a brand check so that the isGenerator works for transpiled generator using a different regenerator runtime versions, such as those from a third party lib.

For this reason I suggest to keep it.